macros/commands/dump_table.sql

```diff
@@ -18,7 +18,7 @@
     {% endif %}
 
     {% set query %}
-        select {{ elementary.escape_select(column_names) }} from {{ relation }}
+        {{ elementary.dedup_by_column_query("unique_id", column_names, relation) }}
```

Comment:
Can you add a `dedup=False`parameter and pass `dedup=True` argument in the sync?
The deduping is unexpected behavior that should be explicitly requested in this macro.